.github/CONTRIBUTING.md

```diff
@@ -50,11 +50,15 @@ If you're not sure what any of that means, check out Thinkful's [GitHub Pull Req
 
 - Limit your changes to only what is required to implement the fix or feature. In particular, avoid style or formatting tools that may modify the formatting of other areas of the code.
 
+- Make sure the code you submit compiles and runs without issues. When we set up unit tests and continuous integration we also expect that the pull request should pass all tests.
+
 - Follow our coding conventions, which we've intentionally kept quite minimal.
 
 ### Coding Conventions
 
-- For variables we use readable camel case: `doSomethingCool`. If they are class members use the 'm_' prefix: `m_DoSomethingCool`. When they are static use the 's_' prefix: `s_DoSomethingCool`.
+- For variables we use readable camel case: `variableName`. If they are class members use Hungarian notation (the 'm_' prefix): `m_ClassMemberVariableName`. When they are static use the 's_' prefix: `s_ClassStaticVariableName`. For class names we use Pascal case: `ClassName`. For functions we also use Pascal case: `FunctionName`, and we use camel case for parameters: `parameterName`.
+
+- For macros we use snake case: `MACRO_NAME` and if it is specifically related to Hazel we add the prefix 'HZ_' to it: `HZ_MACRO_NAME`.
```

Comment:
This is nice, but I think this is a bit too long for one bullet point... And I think the macros we can add in the same bullet, as it is also about the naming?

- Naming convention:
    - For functions we use pascal case: **`FunctionName`**.
    - For (scoped) variables and function parameters we use camel case: **`variableName`** and **`parameterName`**.
    - For class names we use pascal case: **`ClassName`**.
    - For class variables we use the Hungarian notation:
        - Class member variables get the 'm_' prefix: **`m_ClassMemberVariableName`**.
        - Class static variables get the 's_' prefix: **`s_ClassStaticVariableName`**.
    - For macros we use snake case: **`MACRO_NAME`**.
        - If it is specifically related to Hazel, we add the 'HZ_' prefix: **`HZ_MACRO_NAME`**.
        - If there is a macro for the application and for the engine, we add an additional 'CORE_' prefix to the engine macro:  **`HZ_CORE_MACRO_NAME`**.

Reply:
Yes, this works much better. I've changed it as you suggested.